pulp_plugin_template/tests/functional/api/test_download_content.py

```diff
@@ -86,18 +86,19 @@ def test_all(self):
         distribution = client.post(DISTRIBUTION_PATH, body)
         self.addCleanup(client.delete, distribution['_href'])
 
-        # Pick a content unit, and download it from both Pulp Fixtures…
-        unit_path = choice(get_plugin_template_content_paths(repo))
-        fixtures_hash = hashlib.sha256(
-            utils.http_get(urljoin(PLUGIN_TEMPLATE_FIXTURE_URL, unit_path))
-        ).hexdigest()
+        # Pick a content unit (of each type), and download it from both Pulp Fixtures…
+        unit_paths = [choice(paths) for paths in get_plugin_template_content_paths(repo).values()]
+        fixtures_hashes = [hashlib.sha256(
+            utils.http_get(urljoin(PLUGIN_TEMPLATE_FIXTURE_URL, unit_path[0]))
+        ).hexdigest() for unit_path in unit_paths]
 
         # …and Pulp.
         client.response_handler = api.safe_handler
 
-        unit_url = cfg.get_hosts('api')[0].roles['api']['scheme']
-        unit_url += '://' + distribution['base_url'] + '/'
-        unit_url = urljoin(unit_url, unit_path)
+        unit_base_url = cfg.get_hosts('api')[0].roles['api']['scheme']
```

Comment:
Copy this change https://github.com/pulp/pulp_file/pull/169/files

Reply:
I copied it in the other PR. Let's rebase this afterwards.